### DISCUSSION TITLE
How to add a label in a dynamic grouping

### DISCUSSION THREAD

**Discussion by navinleon** [xyflow/xyflow#4079] [Q&A]
Hi,

I'm trying to implement the dynamic grouping, and I want to label the grouped nodes; I can't find the right examples or docs. Can you please help me with an example? or link to the right documentation?
https://pro.reactflow.dev/examples/react/dynamic-grouping
![image](https://github.com/xyflow/xyflow/assets/1019064/bdf08287-96b1-43f8-b762-75a88a806e11)

**Comment by navinleon** [comment]
Figured out by adding the label in GroupNodes component.

**Reply by moklick** [reply]
Perfect! For other people who find this discussion: the group node is just a custom node in this case. Every node can be a group node and you can render anything you want inside a node.
